File: code_snippets/python-sdk/model-upload/predict-first-model.py
from clarifai.client import Model

# Set PAT as an environment variable
#   export CLARIFAI_PAT=YOUR_PAT_HERE # Unix-Like Systems
#   set CLARIFAI_PAT=YOUR_PAT_HERE  # Windows

# Initialize with your model URL
model = Model(url="https://clarifai.com/alfrick/docs-demos/models/my-custom-model")

for response in model.generate("Yes, I uploaded it! "):
    print(response)

File: code_snippets/python-sdk/model-upload/predict-first-model.js
import { Model } from "clarifai-nodejs";

const model = new Model({
  url: "https://clarifai.com/alfrick/docs-demos/models/my-custom-model",
  authConfig: {
    pat: process.env.CLARIFAI_PAT,
  },
});

const response = await model.generate({
  // see available methodNames using model.availableMethods()
  methodName: "generate",
  text1: "Yes, I uploaded it!",
});

console.log(JSON.stringify(response));

// get response data from the response object
Model.getOutputDataFromModelResponse(response);